CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
-* Fixed an issue where ** lints** did not pull docker images for powershell.
+* Fixed an issue where **init** offered a non relevant pack support type.
+* Fixed an issue where **lint** did not pull docker images for powershell.
```

Comment:
are you adding this for a different PR? cause it is not seen here.

Reply:
fixed my typo from a previous one.